src/SurfaceFluxes.jl

```diff
@@ -535,11 +549,11 @@ function momentum_exchange_coefficient(
     κ = SFP.von_karman_const(param_set)
     transport = UF.MomentumTransport()
     FT = eltype(L_MO)
-    ρ_in = TD.air_density(thermo_params, ts_in(sc))
-    ρ_sfc = TD.air_density(thermo_params, ts_sfc(sc))
-    Δρ = ρ_in - ρ_sfc
+    DSEᵥ_sfc = compute_DSEᵥ(param_set, sc, ts_sfc(sc), z_sfc(sc))
+    DSEᵥ_in = compute_DSEᵥ(param_set, sc, ts_in(sc), z_in(sc))
+    ΔDSEᵥ = DSEᵥ_in - DSEᵥ_sfc
     ustar = compute_ustar(param_set, L_MO, sc, uft, scheme)
-    if Δρ ≈ FT(0)
+    if ΔDSEᵥ ≈ FT(0)
         Cd = (κ / log(Δz(sc) / z0(sc, transport)))^2
```

Comment:
Can you please make sure this is the correct neutral limit of your similarity functions?